src/components/Counter.jsx

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react';
```

Comment:
from React 17 you do not need to import React

Reply:
- Delete import React where necessary